packages/@expo/cli/src/start/platforms/PlatformManager.ts

```diff
@@ -68,7 +68,13 @@ export class PlatformManager<
     const redirectUrl = this.props.getRedirectUrl();
     if (redirectUrl) {
       // If the redirect page feature is enabled, check if the project has a resolvable native identifier.
-      const applicationId = await this._getAppIdResolver().getAppIdAsync();
+      let applicationId;
+      try {
+        applicationId = await this._getAppIdResolver().getAppIdAsync();
+      } catch {
+        // do nothing, expected if the applicationId is not defined anywhere (e.g. managed project
```

Comment:
might be worth warning here? otherwise i'm not sure how the developer can discover why this is happening